.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Thanks for submitting a Pull Request!
+
+Please shortly explain your contribution, and if fixing an issue from the tracker, please add "Fixes #XXX", replacing XXX with the issue number.
+
+Be ensure that your contribution follows the [Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/).
+
+Be sure to go over each item in the list below before submitting your pull request.
+-->
+
+### Description
+
+Add your description here
+
+### Checklist
+
+- [ ] Make sure that your commit messages include a [DCO]( https://discourse.ros.org/t/starting-to-enforce-developer-certificate-of-origin-dco-for-some-ros-2-repos/7420).
+- [ ] Make sure your commit messages follow [good practices]( https://chris.beams.io/posts/git-commit/).
+- [ ] Make sure your PR is minimal. Unrelated changes should be submitted separately.
+- [ ] If you have changed how any of the APIs worked, or added new APIs, please update the API documentation.
+- [ ] If you've added a new feature or new package, make sure to update the documentation at https://index.ros.org/doc/ros2/ describing the new feature.
+- [ ] Add or update tests corresponding to your PR.  If tests cannot be added for some reason, be sure to explain why in the description.
+- [ ] While waiting for someone to review your PR, please consider reviewing [another open pull request](https://github.com/ros2/rcl/pulls) to support the maintainers of RCL. Refer to the [ROS code review guide](https://github.com/rosin-project/ros_code_review_guide/blob/master/README.md) for general reviewing guidelines and [Quality Guide: Ensuring code quality](https://index.ros.org/doc/ros2/Contributing/Quality-Guide/) for ROS 2 specific guidelines
+
```

Comment:
As @dirk-thomas pointed out earlier, we should also have a bullet (early in the list) about passing tests locally.